lib/note-editor.jsx

```diff
@@ -251,7 +242,9 @@ export const NoteEditor = React.createClass({
           <div
             style={printStyle}
             className="note-print note-detail-markdown"
-            dangerouslySetInnerHTML={{ __html: noteContent }}
+            dangerouslySetInnerHTML={{
+              __html: markdownEnabled ? renderNoteToHtml(content) : content,
```

Comment:
I noticed that my malicious note would be rendered quickly when printing (if markdown was disabled for the note), maybe we should always pass this through `renderNoteToHtml` for safety's sake?

Reply:
oh you know what…we shouldn't be setting `__html` to `content` if Markdown is disabled. it's not HTML so it shouldn't be treated that way - this is a separate issue so let's do it in another PR (because we'll have to make sure the styling all works out

```js
{ markdownEnabled ? (
	<div
		style={printStyle}
		className="note-print note-detail-markdown"
		dangerouslySetInnerHTML={{ __html: renderNoteToHtml( content ) }}
	/>
) : (
	<div style={printStyle} className="note-print note-detail">{ content }</div>
) }
```